storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/dataaccess/CombinedKvStoreDao.java

```diff
@@ -378,6 +378,22 @@ public Map<String, Long> getColumnCounts() {
     return columnCounts;
   }
 
+  @Override
+  public Map<String, Long> getBlobsSidecarColumnCounts() {
+    final Map<String, Long> columnCounts = new LinkedHashMap<>();
+
+    schema.getColumnMap().entrySet().stream()
+        .filter(
+            stringKvStoreColumnEntry ->
+                List.of(
```

Comment:
I'd put this list to one constant in one place, if possible

Reply:
problem is that those schema columns names are in various places currently .... I'll leave it there for now, we should have unified name inventory one day..